buster/completers/base.py

```diff
@@ -43,6 +43,9 @@ def generate_response(self, system_prompt, user_input) -> Completion:
         logger.info(f"{user_input=}")
         try:
             completion = self.complete(prompt=prompt, **self.completion_kwargs)
+        except openai.error.InvalidRequestError:
+            logger.exception("Invalid request to OpenAI API. See traceback:")
+            return Completion("Something went wrong, try again soon!", True, "Invalid request made to openai.")
```

Comment:
overlap with the previous PR?

Reply:
yes i forgot this bit from the last PR

Reply:
these configs are also starting to get disorganized... might be time to switch over to omegaconf or to do a rehaul